tests/cases/compiler/interfaceInGenericFunctionOuterParameterVarianceMeasurement.ts

```diff
@@ -0,0 +1,17 @@
+// @strict: true
+function f<T>() {
+    interface A<U> {

```

Comment:
This test is mostly just to prove that interfaces don't share the same issue - the `.typeParameters` member we use to calculate variance is already the combined set of local and outer type parameters.